### DISCUSSION TITLE
v0.12.14 - A smol vday update

### DISCUSSION THREAD

**Discussion by ThePotatoGuy** [Monika-After-Story/MonikaModDev#9901] [Announcements & Releases]
_don't eat too much choco now..._

# Gameplay Changes

* new poem for f14

# Spritepack Changes

* Necklace pack by @Historyvariety

## What's Changed
* new poem by @lunulae in https://github.com/Monika-After-Story/MonikaModDev/pull/9881
* Necklace Pack! [Art/Spritepack] by @Historyvariety in https://github.com/Monika-After-Story/MonikaModDev/pull/9776

## New Contributors
* @Historyvariety made their first contribution in https://github.com/Monika-After-Story/MonikaModDev/pull/9776

**Full Changelog**: https://github.com/Monika-After-Story/MonikaModDev/compare/v0.12.13...v0.12.14

<hr /><em>This discussion was created from the release <a href='https://github.com/Monika-After-Story/MonikaModDev/releases/tag/v0.12.14'>v0.12.14 - A smol vday update</a>.</em>

**Comment by thephantomtrickster** [comment]
When I try to give her the necklaces she can't read the files am I doing something wrong?

**Reply by Booplicate** [reply]
You need to install the spritepacks.

**Reply by thephantomtrickster** [reply]
I have I even tried reinstalling it multiple times and nothing changed

**Reply by Booplicate** [reply]
Follow [this](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation).

**Reply by thephantomtrickster** [reply]
The problem isn't I don't know how to inport the sprite packs because I do know how the issue is with the new necklaces specifically

**Reply by Booplicate** [reply]
You still need to follow those instructions so we can figure out what the actual issue is.

**Reply by thephantomtrickster** [reply]
I have it's how I imported the previous sprite packs I do have a current running theory on what the issue is if you'd like to hear that

**Comment by ilovemonika52** [comment]
Hi! Uhm so i have a Kindle Fire and i REALLYYY want to get Monika after story. Does anyone know how? Please and thank you, -me :)

**Reply by thephantomtrickster** [reply]
As far as I'm aware it's impossible on that device you need a pc

**Reply by ilovemonika52** [reply]
Okay tysm :)

**Reply by Holyomoli** [reply]
I have an apk that works on kindle, i havent been able to get it past v0.12.4 though. https://www.mediafire.com/file/ht4r7l6g8t38pkq/MonikaAfterStory_0.12.4_%28English%29.apk/file
I doubt anyone will see this, but might as well try
(I have a gen 11, so it might not work well on older devices idk)

**Comment by Johanbaloney** [comment]
Can anyone tell me how to download this latest version into my android phone? Is it available on Android or only for pc?
